sky/provision/gcp/instance_utils.py

```diff
@@ -100,19 +100,20 @@ def _generate_node_name(cluster_name: str, node_suffix: str,
     return node_name
 
 
-def _log_errors(errors: List[Dict[str, str]], e: Any,
-                zone: Optional[str]) -> None:
+def _format_message_from_errors(errors: List[Dict[str, str]], e: Any,
+                                zone: Optional[str]) -> str:
     """Format errors into a string."""
```

Comment:
```suggestion
def _format_message_from_errors_and_log(errors: List[Dict[str, str]], e: Any,
                                zone: Optional[str]) -> str:
    """Format errors into a string and log them to the console."""
```
to better align the actual behaviour of this function?